src/customers/account-dashboard-my-downloadable-products.md

```diff
@@ -7,6 +7,16 @@ The My Downloadable Products page links to each order of downloadable products.
 ![]({% link images/images-ee/customer-account-my-downloadable-products.png %}){: .zoom}
 _My Downloadable Products_
 
+The following table describes the _My Downloadable Products_ values:
+
+|Column|Description|
+|--- |--- |
+|Order#|The [order]({% link /sales/orders.md %}) in which the downloadable product was purchased.|
+|Date| Order creation date, which provides a link to the order detail.|
```

Comment:
This was my fault because I added my suggestion to the wrong line before. It is the Order# that has the link, not the date. Sorry about that.

```suggestion
|Order#|The [order]({% link /sales/orders.md %}) in which the downloadable product was purchased. Provides a link to the order detail.|
|Date| Order creation date.|
```